ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/ActivePeers.hs

```diff
@@ -115,8 +116,10 @@ belowTargetBigLedgerPeers actions
                           inProgressPromoteWarm = inProgressPromoteWarm
                                                <> selectedToPromote
                         },
-        decisionJobs  = [ jobPromoteWarmPeer actions policy peeraddr IsBigLedgerPeer peerconn
-                        | (peeraddr, peerconn) <- Map.assocs selectedToPromote' ]
+        decisionJobs  = [ jobPromoteWarmPeer actions policy peeraddr IsBigLedgerPeer isLocalRootPeer peerconn
+                        | (peeraddr, peerconn) <- Map.assocs selectedToPromote'
+                        , let isLocalRootPeer = LocalRootPeers.member peeraddr localRootPeers
```

Comment:
We need to check if the peer is a trusted local root peer; the same in other places.

Reply:
I think checking local root is enough because when the information is going to be used the peer will guaranteed be a trusted one. As far as consensus cares it just wants to know if it is local root or external peer. So I think it is enough to check if it is local root

Reply:
You're right it's enough for us to check that it's a local root peer.